src/toast/util/toastHooks.ts

```diff
@@ -0,0 +1,83 @@
+import React, {useCallback} from "react";
+
+import {generateRandomString} from "../../core/utils/string/stringUtils";
+import {ToastContext} from "../ToastProvider";
+import {ToastData} from "./toastTypes";
+
+/**
+ * @returns {Object} Current value of ToastContext
+ */
+function useToastContext() {
+  const context = React.useContext(ToastContext);
+
+  if (context === undefined) {
+    throw new Error("No context found for ToastContext");
```

Comment:
```suggestion
    throw new Error("Trying to consume ToastContext outside of its provider.");
```

Reply:
Can we update Toggle Error message also like this?

Reply:
Yep